### PR TITLE
Generate mocks for image and image index

### PR DESCRIPTION
**Pull request by glyn**

### CODE REVIEW

File: pkg/v1/image.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 )
 
 // Image defines the interface for interacting with an OCI v1 image.
+//go:generate counterfeiter -o fake/FakeImage.go . Image
```
Comment (jonjohnsonjr) [inline pkg/v1/image.go]: I think snake_case is the standard convention for golang files names? Just: `image.go`

File: pkg/v1/index.go [hunk 2]
```diff
@@ -19,6 +19,7 @@ import (
 )
 
 // ImageIndex defines the interface for interacting with an OCI image index.
+//go:generate counterfeiter -o fake/FakeImageIndex.go . ImageIndex
```
Comment (jonjohnsonjr) [inline pkg/v1/index.go]: Same here: `image_index.go`

**Comment by glyn** [comment]
> LGTM other than filename nits. I'm assuming you can control that, but I haven't dug into counterfeiter so maybe they need to be PascalCase?

No, my mistake. I'll fix that.

File: pkg/v1/image.go (hunk 1 above)
Comment (glyn) [reply pkg/v1/image.go]: Fixed.

File: pkg/v1/index.go (hunk 2 above)
Comment (glyn) [reply pkg/v1/index.go]: Fixed.

**Comment by jonjohnsonjr** [comment]
Thanks!

**Pull request merged**